controllers/authController.js

```
@@ -74,6 +77,29 @@ exports.login = catchAsync(async (req, res, next) => {
 	createSendToken(user, StatusCodes.OK, res);
 });
 
+exports.logout = catchAsync(async (req, res, next) => {
+	let token;
+	if (req.headers.authorization) {
+		token = req.headers.authorization.split(' ')[1];
+	} else if (req.body.token) {
+		token = req.body.token;
+	} else if (req.cookies.jwt) {
+		token = req.cookies.jwt;
+	}
+
+	if (!token)
+		return next(new AppError('No token provided', StatusCodes.BAD_REQUEST));
+
+	const deletedRecords = await delAsync(token);
+
+	if (!deletedRecords)
+		return next(new AppError('User already logout', StatusCodes.NOT_FOUND));
```

I'm not sure if you should throw not found here. 
As a user, I wanted to be logged out, if I'm already logged out this satisfied my request, so I guess you could respond with `204 No content` or just `200 Ok`

What do you think @MoodupDev

---

@arkus7 yes I agree